src/ducks/settings/constants.js

```diff
@@ -5,6 +5,7 @@ export const DEFAULTS_SETTINGS = {
   _type: 'io.cozy.bank.settings',
   _id: 'configuration',
   id: 'configuration',
+  balancesNotifications: {},
```

Comment:
can you update the cozy-doctypes repo if you store more information?

Reply:
yes it is planned, now, here => https://github.com/cozy/cozy-doctypes/pull/230